src/test/kotlin/org/rust/ide/annotator/fixes/ConvertToSizedTypeFixTest.kt

```diff
@@ -5,33 +5,33 @@
 
 package org.rust.ide.annotator.fixes
 
-import org.rust.ide.annotator.RsAnnotatorTestBase
+import org.rust.ide.annotator.RsAnnotationTestBase
 
-class ConvertToSizedTypeFixTest : RsAnnotatorTestBase() {
+class ConvertToSizedTypeFixTest : RsAnnotationTestBase() {
 
-    fun `test convert function arg to reference`() = checkQuickFix("Convert to reference", """
-        fn foo(slice: [u8]/*caret*/) {}
+    fun `test convert function arg to reference`() = checkFixByText("Convert to reference", """
+        fn foo(slice: <error>[u8]/*caret*/</error>) {}
     """, """
         fn foo(slice: &[u8]) {}
     """)
 
-    fun `test convert function return type to reference`() = checkQuickFix("Convert to reference", """
-        fn foo() -> [u8]/*caret*/ { unimplemented!() }
+    fun `test convert function return type to reference`() = checkFixByText("Convert to reference", """
+        fn foo() -> <error>[u8]/*caret*/</error> { unimplemented!() }
     """, """
         fn foo() -> &[u8] { unimplemented!() }
     """)
 
-    fun `test convert function arg to Box`() = checkQuickFix("Convert to Box", """
+    fun `test convert function arg to Box`() = checkFixByText("Convert to Box", """
         trait Foo {}
-        fn foo(foo: Foo/*caret*/) {}
+        fn foo(foo: <error>Foo/*caret*/</error>) {}
     """, """
         trait Foo {}
         fn foo(foo: Box<Foo>) {}
     """)
 
-    fun `test convert function return type to Box`() = checkQuickFix("Convert to Box", """
+    fun `test convert function return type to Box`() = checkFixByText("Convert to Box", """
         trait Foo {}
-        fn foo() -> Foo/*caret*/ { unimplemented!() }
+        fn foo() -> <error>Foo/*caret*/</error> { unimplemented!() }
```

Comment:
It looks a bit weird that now we have to specify both error and caret...

Reply:
See [this](https://github.com/intellij-rust/intellij-rust/pull/2673#discussion_r204208425) comment